src/beast/site_scons/site_tools/VSProject.py

```diff
@@ -127,6 +127,11 @@ def gen():
             if type(item) == dict:
                 for k, v in xsorted(item.items()):
                     yield k + '=' + v
+            elif type(item) == tuple:
+                if len(item) != 2:
+                    raise ValueError ("Tuple shoud have exactly two elements: " +
+                                      str(item))
+                yield item[0] + '=' + item[1]
```

Comment:
Tiny quibble here.  If you are gluing together more than two items, you get better efficiency using `%`:  `yield '%s=%s' % item` - because it doesn't have to create two intermediate strings...!